Tests/CryptoSwiftTests/AESOCBTests.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  CryptoSwift
+//
+//  Copyright (C) Marcin Krzyżanowski <marcin@krzyzanowskim.com>
+//  This software is provided 'as-is', without any express or implied warranty.
+//
+//  In no event will the authors be held liable for any damages arising from the use of this software.
+//
+//  Permission is granted to anyone to use this software for any purpose,including commercial applications, and to alter it and redistribute it freely, subject to the following restrictions:
+//
+//  - The origin of this software must not be misrepresented; you must not claim that you wrote the original software. If you use this software in a product, an acknowledgment in the product documentation is required.
+//  - Altered source versions must be plainly marked as such, and must not be misrepresented as being the original software.
+//  - This notice may not be removed or altered from any source or binary distribution.
+//
+
+import XCTest
+@testable import CryptoSwift
+
+class OCBTests: XCTestCase {
+
+
+  struct TestFixture {
+    let K: Array<UInt8>
+    let N: Array<UInt8>
+    let P: Array<UInt8>
+    let C: Array<UInt8>
+  }
+
+  func testAESOCBWithRFC7253Tests() {
+
+    var fixtures = [
+      TestFixture(K: Array<UInt8>(hex: "000102030405060708090A0B0C0D0E0F"),
+                  N: Array<UInt8>(hex: "BBAA99887766554433221100"),
+                  P: Array<UInt8>(hex: ""),
```

Comment:
This fixture revealed a bug: if the ciphertext is empty (there is only a tag), the tag is not verified

Reply:
Fixed!